compiler/test/fail_compilation/named_arguments_ifti_error.d

```diff
@@ -0,0 +1,20 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/named_arguments_ifti_error.d(17): Error: template `f` is not callable using argument types `!()(int, int)`
```

Comment:
This error message does not seem to accurately explain what the problem is. I would expect something along the lines of: "Missing argument for parameter 'y'".

Reply:
The problem is that the current template instantiation code gags / discards most error messages, requiring a refactor to make more informative errors. I don't want to add that to this PR, but want to look into improving this next (which is long overdue; template errors are a big source of confusion especially with `ref` and lambda parameters).